app/src/main/java/io/zenandroid/onlinego/model/ogs/OGSChallengeRequest.kt

```diff
@@ -4,30 +4,27 @@ import io.zenandroid.onlinego.ogs.TimeControl
 
 data class OGSChallengeRequest (
         val initialized: Boolean,
-        val min_ranking: Int = -1000,
-        val max_ranking: Int = 1000,
+        val min_ranking: Int = 0,
```

Comment:
Why change these, are they creating problems?

Reply:
Ranked=true was being ignored/reverted to false by the server. One reason is that matches were private by default, and a ranked match can't be private;
Then I still needed to change the ranks to have the Ranked property accepted to True: I'm not really sure why, maybe the server expects a real range on ranked eligibility calculations

Reply:
One thing I didn't implement is the auto-setting of Ranked to No when Private is set to Yes (also no longer allowing to edit Ranked), like OGS does; didn't find an easy way to do it.